e2e/heapIgnore.spec.js

```diff
@@ -54,6 +54,23 @@ describe('HeapIgnore', () => {
     assert(JSON.stringify(res)).not.match(/Foo;\|/)
   });
 
+  it(':android: should ignore the interaction', async () => {
+    // Get all the events from redis, and assert that none of the requests match the ignored
+    // interaction.
+    const { err, res } = await new Promise((resolve, reject) => {
+      testUtil.findAndroidEventInRedisRequests({}, (err, res) => {
```

Comment:
What do you think about doing this switch within a separate fn?  So make something like `findEvent({})` that does the appropriate switch based on the running platform?  That way we don't have to duplicate the rest of the test.  I _believe_ you get access to that as a global while detox is running...